zerver/lib/actions.py

```diff
@@ -1457,6 +1457,13 @@ def check_send_stream_message(sender, client, stream_name, topic, body):
 
     return do_send_messages([message])[0]
 
+def check_send_private_message(sender, client, body):
+    # type: (UserProfile, Client, Text) -> int
+    addressee = Addressee.for_user_profile(sender)
```

Comment:
I think you want to pass in the user_profile of the person receiving the message here.